http-echo.yaml

```diff
@@ -49,8 +49,37 @@ update:
     strip-prefix: v
 
 test:
+  environment:
+    contents:
+      packages:
+        - curl
   pipeline:
-    # AUTOGENERATED
     - runs: |
         http-echo --version
         http-echo --help
+
+        # Start server in background with test message
+        http-echo -listen=:8080 -text="test message" &
```

Comment:
Let's use `test/daemon-check-output` here and put all of other `curl` commands to `post:` section.